packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -126,9 +128,14 @@ const useCoinmarketVerifyAccount = ({
         [accounts, currency, device, isDebug, symbol],
     );
 
+    const { supportedMainnets, supportedTestnets } = useNetworkSupport();
+    const findNetworkByType = (network: Network) => network.networkType === currency;
```

Comment:
`currency` is `CryptoId` type, this will not work. You can use the function `cryptoIdToNetwork(currency)` it will return `Network` type and then use it in that function above. But I am not sure is the way. :)

LTC
```
networkType='bitcoin'
currency='litecoin'
```
POS
```
networkType='ethereum'
currency='ethereum--0x455e53cbb86018ac2b8092fdcd39d8444affc3f6'
```

Reply:
done